aliveim.go

```diff
@@ -56,7 +62,11 @@ func handleAlivePost(rw http.ResponseWriter, request *http.Request) {
 	} else {
 		timer := time.NewTimer(time.Millisecond * time.Duration(aliverequest.Timeout))
 		deviceTimer := DeviceTimer{aliverequest.DeviceID, timer}
+
+		mutex.Lock()
 		timersMap[aliverequest.DeviceID] = deviceTimer
+		mutex.Unlock()
+
 		go deviceTimer.startTimer()
```

Comment:
I'm reviewing this again, unless we can reproduce a race condition I'd go merging this as it's already a good progress towards a better concurrency model.

So ACK from me